src/rev_ai/__init__.py

```diff
@@ -1,4 +1,4 @@
 # -*- coding: utf-8 -*-
 """Top-level package for rev_ai"""
 
-__version__ = '0.0.4'
+__version__ = '0.1.0'
```

Comment:
As Josh noted, these are breaking changes

Reply:
Josh is correct. But I think this requires a bump in the major version.
```suggestion
__version__ = '1.0.0'
```

Relevant Docs: https://semver.org/

> - MAJOR version when you make incompatible API changes,
> - MINOR version when you add functionality in a backwards-compatible manner, and
> - PATCH version when you make backwards-compatible bug fixes.

Reply:
ask @JDongian for confirmation.